units/event_metajob.go

```diff
@@ -302,19 +146,26 @@ func (j *eventMetaJob) Run(ctx context.Context) {
 	if j.limit <= 0 {
 		j.limit = evergreen.DefaultEventProcessingLimit
 	}
-	j.events, err = event.FindUnprocessedEvents(j.limit)
+	j.events, err = event.FindUnprocessedEvents(-1)
```

Comment:
why not lift all of this into a Populate() function in crons, it doesn't seem like generating the jobs is really going to take that long and this just seems confusing.

Reply:
Done. The event-metajob is no more!